### PR TITLE
Delete v1 connectors

### PR DESCRIPTION
**Pull request by alexesprit**
More cleanup. All these connectors have v2 implementations which works fine ~~(except Hillydilly)~~.

**Update:**
Blinkbox Music and TTNET Muzik are no longer available. The first one was closed, the second one redirects to new site which has another layout.

### CODE REVIEW

**Comment by ex47** [comment]
:+1:

**Comment by ex47** [comment]
maybe you could also address #1022

**Comment by alexesprit** [comment]
What you mean?

**Comment by ex47** [comment]
Drop [this](https://github.com/david-sabata/web-scrobbler/blob/a4944bca28edd535a16acbcbfc4c783cbff6b5be/core/connectors.js#L985-L990), [this](https://github.com/david-sabata/web-scrobbler/blob/a4944bca28edd535a16acbcbfc4c783cbff6b5be/core/connectors.js#L237-L242) and this [file](https://github.com/david-sabata/web-scrobbler/blob/ff547dfdfd1b6682f61baf50ed4bdab0516106e8/connectors/v2/spotify.js). As a part of cleanup.

**Comment by alexesprit** [comment]
Evropa's URLs are both valid (in #1022 I mistyped and went to wrong URL). These URLs need to be merged into one item in the connectors list. I'll make a new PR for this case.

Old Spotify connector is useless, but it's in v2 format. Will do it in another PR. In this PR I deleted connectors in v1 format.

File: manifest.json [hunk 1]
```diff
@@ -15,7 +15,6 @@
    "web_accessible_resources": [
       "icon128.png",
       "connectors/jango-dom-inject.js",
```
Comment (inverse) [inline manifest.json]: Shouldn't this `jango` one be removed too?

**Comment by inverse** [comment]
Awesome work :+1:

File: manifest.json (hunk 1 above)
Comment (alexesprit) [inline manifest.json]: Yes, that need to be cleaned too.

**Comment by inverse** [comment]
Thanks!

**Pull request merged**